package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@google/maps": "0.4.3",
     "body-parser": "1.17.2",
-    "cors": "2.8.4",
+    "cors": "^2.8.4",
```

Comment:
@Sanderson239 please remove the `^` here so as to fix this package's version, thanks!